webui/templates/500.html

```diff
@@ -7,10 +7,10 @@
   <div class="row my-5">
     <div class="col-12 col-sm-4 offset-sm-4">
       <div class="card">
-        {# XXX Should this page has a png image here? #}
-        {#<img src="{{ url_for('static', filename='orcid.png') }}" class="card-img-top" alt="ORCID Logo">#}
+        <img src="{{ url_for('static', filename='404.png') }}" class="card-img-top" alt="">
```

Comment:
Yeah, cause it shows it can't find, same as 404, though the error type is slightly different, the user's still "searching"...? if that makes sense. We can choose something different, though, if desired.

Reply:
I would like to have something distinct since the two kinds of errors are very different. 404 is "you asked for something that doesn't exist", whereas 500 is "whoops, server had a problem and we couldn't do that, please try again later".

I'll merge this, but please keep this in mind to update later.